src/debug/debug.cpp

```diff
@@ -2308,7 +2333,13 @@ bool CDebugVar::SaveVars(char *name)
 	if (varList.size() > 65535) return false;
 
 	FILE* f = fopen(name,"wb+");
-	if (!f) return false;
+	if (!f) {
+		DEBUG_ShowMsg("DEBUG: Output of vars failed.\n");
+		return false;
+	}
+	DEBUG_ShowMsg("DEBUG: vars file %s created.\n",
```

Comment:
When showin a path to any user-readable location, I think we should surround it with a single quotes, like this: `'%s'`. This makes it more readable if the path happens to include whitespace characters.

We should be more strict about this, I see some paths not surrounded by quotes in our log.